packages/dht/src/connection/WebRTC/WebRtcConnector.ts

```diff
@@ -28,6 +28,15 @@ import { PortRange } from '../ConnectionManager'
 
 const logger = new Logger(module)
 
+const replaceIpIfCandidateTypeIsHost = (candidate: string, ip: string) => {
```

Comment:
What it means if the type is `host`? E.g. that we have an IP address, but it is an internal IP address?

If that's the case, maybe this method could be called `withExternalpAdress`?

Reply:
When ICE connectivity checks are started, the peers will find any IP addresses that the host machine has on its interfaces. The IceCandidates created with these IP addresses always have the type host.

Reply:
If we use terminology of internal/external:
- the "IP addresses that the host machine has on its interface" are "internal" IPs?
- and in this function we rename internal IP with an external IP?